be-java-springboot/files/metadata.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: OpenJDK
-description: "OpenJDK is a free and open-source implementation of the Java Platform, Standard Edition. Technologies: Spring Boot 2.1, OpenJDK 11"
-supplier: https://adoptopenjdk.net
+description: "OpenJDK is a free and open-source implementation of the Java Platform, Standard Edition. Technologies: Spring Boot 2.1, OpenJDK 17"
```

Comment:
@michaelsauter I have updated this file, but I'm sure why we have it. For which purpose this file is used?

Reply:
@metmajer Is there documentation on this?

I believe this to be used in document generation as a description of the component.

Reply:
@stitakis @michaelsauter yes, this information will be used in LeVA documentation. If this says OpenJDK 17, it must correspond with what's actually used.